tests/validation_v2/utils/gce-10acre-ranch/gce-10acre-ranch

```diff
@@ -0,0 +1,387 @@
+#!/bin/bash
```

Comment:
We can't copy this script here. It should be downloaded in the bootstrap script and every else can assume it is on the PATH.